src/services/calcService.ts

```diff
@@ -0,0 +1,18 @@
+let yearly: number;
+let monthly: number;
+
+export const yearlyPremium = (carValue: number, riskRating: number): number => {
+  //This if statement makes sure that invalid numbers cannot be submitted
+  if (!(carValue > 0 && riskRating > 0)) {
```

Comment:
Same comment as the JS file. An OR || could be helpful to further prevent the unwanted input?